frontend/src/relay/auth.tsx

```diff
@@ -78,14 +83,15 @@ export const ExternalLink: React.FC<ExternalLinkProps> = ({
         <input type="hidden" name="jwt" />
 
         <button
+            {...{ ref }}
             className={className}
```

Comment:
Then you can also combine it with the next line 
```suggestion
            {...{ ref, className }}
```